test.js

```diff
@@ -4,14 +4,41 @@ import inRange from 'in-range';
 import fn from './';
 
 test('promise', async t => {
+	t.plan(1);
 	const end = timeSpan();
 	await fn(50);
-	t.true(inRange(end(), 30, 70));
+	t.true(inRange(end(), 30, 70), 'should be delayed');
 });
 
 test('thunk', async t => {
+	t.plan(2);
```

Comment:
I don't see the point in using planned assertions here as the only way the assertion are not reached if the promise never resolve, which planned assertions won't help with, or if it rejects, which is already handled by AVA.